cmd/helm/install.go

```diff
@@ -119,7 +119,7 @@ func (i *installCmd) run() error {
 		helm.ValueOverrides(rawVals),
 		helm.ReleaseName(i.name),
 		helm.InstallDryRun(i.dryRun),
-		helm.InstallReuseName(i.reuseName),
+		helm.InstallReuseName(i.replace),
```

Comment:
Should all places of reuse name be `replace`? It it not in any release and might be more clear down the road.  Not sure what is best for the protos.

Just my 2c